conf/machine/imx8mnevk.conf

```diff
@@ -1,6 +1,6 @@
 #@TYPE: Machine
 #@NAME: NXP i.MX8M Nano DDR4 Evaluation Kit
-#@SOC: i.MX8MM
+#@SOC: i.MX8MN
```

Comment:
Nice to be corrected, but since this is the only change in this file and it is not related to the Jailhouse - maybe it makes sense to drop this from the PR.

I'll deal with this typo later, thanks for pointing it out! 👍

Reply:
Yes, that's right. He stayed after squash. I had to check machine config first since 8mn is supporting jailhouse, so I caught that typo.